ui/src/testing/stubs/cluster-api-stub.ts

```diff
@@ -23,6 +24,7 @@ export class ClusterApiStub extends ClusterApi {
 
   listClusters(extraHttpRequestParams?: any): Promise<ClusterListResponse> {
     return new Promise<ClusterListResponse>(resolve => {
+      console.log(this.cluster);
```

Comment:
nit: if you want to keep this log statement, then it probably should say something about what's happening to this cluster or why it's important.